lib/rubocop/cop/style/bare_percent_literals.rb

```diff
@@ -4,6 +4,14 @@ module RuboCop
   module Cop
     module Style
       # This cop checks if usage of %() or %Q() matches configuration.
+      #
+      # @example
```

Comment:
Configurable cops should have documentation like the one here https://github.com/bbatsov/rubocop/blob/c329653bae8ceaaa46f90baaa83abf1209f416a4/lib/rubocop/cop/naming/method_name.rb

(good and bad examples depend on what's configured)